JASP-Engine/JASP/R/contingencytables.R

```diff
@@ -487,6 +490,9 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
                                 overtitle = ci.label, type = "number", format = "dp:3")
   table$addColumnInfo(name = paste0("up[", fold, "]"),    title = "Upper", 
                                 overtitle = ci.label, type = "number", format = "dp:3")
+  table$addColumnInfo(name = paste0("p[", fold, "]"),     title = "p", 
```

Comment:
can't this be `type = "pvalue"`?

Reply:
yes, it was supposed to be type = "pvalue", thanks for catching it